content/docs/reference/design-docs/timetable/index.en.md

```diff
@@ -246,49 +246,72 @@ Other errors can happen at runtime:
 
 During simulation, **if a target arrival time cannot be achieved, the rest of the schedule still stands**.
 
+The mission model in OSRD is represented almost like a Train Schedule with the addition of 2 fields: 
+- `step: Timestamp (ISO 8601)` corresponds to the delay between each train
+- duration: Timestamp (ISO 8601)` which corresponds to the total duration of the mission.
+
+## Example
+
+A mission with an occurrence of 15 min and a duration of 2 hours will see 8 trains running from the departure time.
```

Comment:
Here you use the term "occurence", but above, you have the term "step". Maybe "step" would be more appropriate here?